### PR TITLE
chore: Add mail contact for private requests

### PR DESCRIPTION
**Pull request by Larsluph**
Add mail option to issue creation page for private requests and users without Discord accounts (either due to geo bans or for personal reasons)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -2,4 +2,7 @@ blank_issues_enabled: true
 contact_links:
   - name: Discord
     url: https://discord.gg/KDQP7fR467
-    about: 'Join the discord server to contact us directly'
+    about: Join the discord server to contact us directly
+  - name: Email
+    url: mailto:vuetorrent@larsluph.dev?cc=dw.daanwijns@gmail.com
```
Comment (WDaan) [inline .github/ISSUE_TEMPLATE/config.yml:7]: We probably should only rely on dedicated Vuetorrent email addresses and not personal ones.

**Review by WDaan — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (Larsluph) [reply .github/ISSUE_TEMPLATE/config.yml:7]: WDYM by dedicated email address? Do you have one we can use?

Comment (WDaan) [reply .github/ISSUE_TEMPLATE/config.yml:7]: Would prefer to not have a personal one in there.

Don't really have an alternative ready right now.

Comment (Larsluph) [reply .github/ISSUE_TEMPLATE/config.yml:7]: So I leave only mine?

Comment (WDaan) [reply .github/ISSUE_TEMPLATE/config.yml:7]: If you're okay with that, yes :)

**Review by WDaan — APPROVED** [review]

**Pull request merged**